libmscore/textbase.h

```diff
@@ -288,6 +298,9 @@ class TextBase : public Element {
       bool deleteSelectedText(EditData&);
 
       void selectAll(TextCursor*);
+      void multiClickSelect(EditData&, MultiClick);
+      bool isPrimed() const               { return _primed; }
```

Comment:
Is this `primed` flag used anywhere? I don't see this flag influence anything currently.

Reply:
It's used in startTextEditingOnMouseRelease in events.cpp

Reply:
Ah, I see now. Doesn't that duplicate the logic of `selected` flag though? That is, would calling `textBase->selected()` instead of `textBase->isPrimed()` achieve the desired effect? Or is the purpose of this flag different?

Reply:
The purpose is different. Without an extra flag startTextEditingOnMouseRelease can't tell the difference between the item being selected (for the first time, in which, we don't want to start editing) or for the second time (in which case we do). That's because the selection status is already set before mouse up. Hence the extra flag.

Reply:
OK, thanks for the explanation! Then I'll leave a probably final code style review, and I believe after correcting those styling issues this PR can be merged.